ui/src-tauri/src/main.rs

```diff
@@ -216,6 +217,13 @@ async fn stop_password_prompt(password_store: State<'_, PasswordStore>) -> Resul
     Ok(())
 }
 
+/// Sends all receiving keys to the front end
+#[tauri::command]
+async fn receiving_keys(config: State<'_, Config>) -> Result<Vec<String>, ()> {
+    let keys = get_receiving_keys(&config.service_url).await?;
+    Ok(keys)
+}
```

Comment:
- Lift the error to the caller (substitute `???` with the right error type)
- Refer to display as UI not front-end since front-end is reserved for dApp front-end.

```suggestion
/// Sends all receiving keys to the UI.
#[tauri::command]
async fn receiving_keys(config: State<'_, Config>) -> Result<Vec<String>, ???> {
    get_receiving_keys(&config.service_url).await
}
```

Reply:
took a slightly different approach since the reqwest error type doesn't implement serialize